tests/artifacts/lib.rs

```diff
@@ -0,0 +1,352 @@
+//! Adaptors between compression crates and Rust's modern asynchronous IO types.
+//!
```

Comment:
What is this file for?

Reply:
it _is_ the test case

Reply:
But this file `tests/artifacts/lib.rs` contains duplicate code from [`src/lib.rs`](https://github.com/Nullus157/async-compression/blob/main/src/lib.rs).

Did you accidentally copied `src/lib.rs` to `testrs/artifacts/lib.rs`?

Reply:
Oh I see, it's the file being compressed.
I thought it was some duplicate code in tests and it got me confused.